apps/studio/components/interfaces/Functions/Functions.templates.ts

```diff
@@ -0,0 +1,98 @@
+export const EDGE_FUNCTION_TEMPLATES = [
+  {
+    value: 'hello-world',
+    name: 'Simple Hello World',
+    description: 'Basic function that returns a JSON response',
+    content: `// Setup type definitions for built-in Supabase Runtime APIs
+import "jsr:@supabase/functions-js/edge-runtime.d.ts";
+interface reqPayload {
+  name: string;
+}
+
+console.info('server started');
+
+Deno.serve(async (req: Request) => {
+  const { name }: reqPayload = await req.json();
+  const data = {
+    message: \`Hello \${name} from foo!\`,
+  };
+
+  return new Response(
+    JSON.stringify(data),
+    { headers: { 'Content-Type': 'application/json', 'Connection': 'keep-alive' }}
+  );
+});`,
+  },
+  {
+    value: 'database-access',
+    name: 'Supabase Database Access',
+    description: 'Example using Supabase client to query your database',
+    content: `// Setup type definitions for built-in Supabase Runtime APIs
+import "jsr:@supabase/functions-js/edge-runtime.d.ts";
+import { createClient } from 'jsr:@supabase/supabase-js@2'
+
+Deno.serve(async (req) => {
+  try {
+    const supabase = createClient(
+      Deno.env.get('SUPABASE_URL') ?? '',
+      Deno.env.get('SUPABASE_ANON_KEY') ?? '',
+      { global: { headers: { Authorization: req.headers.get('Authorization')! } } }
+    )
+
+    const { data, error } = await supabase.from('countries').select('*')
```

Comment:
we've replaced the countries table recently with a colors table so maybe lets use that instead?

Reply:
will replace this with something more generic like "table_name"